src/features/layers_in_area/atoms/areaLayersListResource.ts

```diff
@@ -67,18 +67,20 @@ const areaLayersListDependencyAtom = createAtom(
 );
 
 export const areaLayersListResource = createResourceAtom(async (params) => {
-  if (params?.createLayerFeatureActivated === null) return; // Avoid double request
-  if (!params?.focusedGeometry) return;
+  if (!params) return;
+  if (params.createLayerFeatureActivated === null) return; // Avoid double request
   const body: {
     eventId?: string;
     geoJSON?: GeoJSON.GeoJSON;
     eventFeed?: string;
     appId?: string;
-  } = {
-    geoJSON: params?.focusedGeometry.geometry,
-  };
+  } = params.focusedGeometry
+    ? {
+        geoJSON: params.focusedGeometry.geometry,
```

Comment:
From top areaLayersListDependencyAtom it looks like  focusedGeometry  could be null. 
In that case `params.focusedGeometry.geometry,` will throw error
Maybe we need to process it on top, like it was, or return optional chaining

Reply:
But it's checked in ternary operator just one line above:
if there's no `params.focusedGeometry`, empty object will be returned and we won't try to access `params.focusedGeometry.geometry`

Reply:
ah, that's all right then!